examples/opensuse.yaml

```diff
@@ -3,7 +3,10 @@ images:
 # Hint: run `limactl prune` to invalidate the "Current" cache
 - location: https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-JeOS.x86_64-15.4-OpenStack-Cloud-Current.qcow2
   arch: "x86_64"
-# No aarch64 OpenStack build found :(
+  digest: "sha256:378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195"
+- location: https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2
```

Comment:
Can't download the file (from Japan)
```console
$ LANG=C wget https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2
--2023-04-06 09:00:49--  https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2
Resolving download.opensuse.org (download.opensuse.org)... 2001:67c:2178:8::13, 195.135.221.134
Connecting to download.opensuse.org (download.opensuse.org)|2001:67c:2178:8::13|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://mirrorcache-jp.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2 [following]
--2023-04-06 09:00:50--  https://mirrorcache-jp.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2
Resolving mirrorcache-jp.opensuse.org (mirrorcache-jp.opensuse.org)... 2400:8902::f03c:93ff:fec8:af18, 139.162.108.149
Connecting to mirrorcache-jp.opensuse.org (mirrorcache-jp.opensuse.org)|2400:8902::f03c:93ff:fec8:af18|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://mirror-jp.firstyear.id.au/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2 [following]
--2023-04-06 09:00:50--  https://mirror-jp.firstyear.id.au/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2
Resolving mirror-jp.firstyear.id.au (mirror-jp.firstyear.id.au)... 2400:8902::f03c:93ff:fec8:afa1, 139.162.108.7
Connecting to mirror-jp.firstyear.id.au (mirror-jp.firstyear.id.au)|2400:8902::f03c:93ff:fec8:afa1|:443... connected.
HTTP request sent, awaiting response... 500 Internal Server Error
2023-04-06 09:00:51 ERROR 500: Internal Server Error.
```

Reply:
I suspect this is a server issue on that particular mirror. Maybe try a different one from https://mirrors.opensuse.org ?

FWIW, I just downloaded the image (from Canada), and it seems to work fine on M1.

Reply:
Filed https://bugzilla.opensuse.org/show_bug.cgi?id=1210240